mopidy/commands.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Not needed anymore?

Reply:
It's actually a new thing in Python 3.7. It allows us in type hints to refer to types defined later in the file, without using strings to refer to them, and it improves performance, as we don't need to import the objects just used for type annotations to run the code. They can be hidden inside `if TYPE_CHECKING` and thus only be imported when mypy or other type checkers are run.